cli/cat.go

```diff
@@ -29,26 +41,35 @@ func (cmd *CatCommand) GetName() string {
 
 // IsSane returns true if command is sane
 func (cmd *CatCommand) IsSane() bool {
-	return cmd.Path != ""
+	return cmd.args.Path != ""
 }
 
 // PrintUsage print command usage
 func (cmd *CatCommand) PrintUsage() {
-	log.UserInfo("Usage:\ncat <secret>")
+	fmt.Println(cmd.Usage())
+}
+
+// Usage returns usage information
+func (cmd *CatCommand) Usage() string {
```

Comment:
I wish I didn't have to repeat the 4 lines of this body everywhere, but there's no method that returns the usage rather than writing it to an `io.Writer`.

Reply:
Yay, I figured it out by moving this to a function and adding a `GetArgs` method to the interface.